deepblast/nw.py

```diff
@@ -55,7 +55,7 @@ def _forward_pass_numba(theta, A):
         for j in range(1, M + 1):
             maxargs[x] = A[i - 1, j - 1] + V[i - 1, j]  # x
             maxargs[m] = V[i - 1, j - 1]  # m
-            maxargs[y] = A[i - j, 1 - 1] + V[i, j - 1]  # y
+            maxargs[y] = A[i - 1, j - 1] + V[i, j - 1]  # y
```

Comment:
Hi, I'm really confused where this is coming from -- yes that is definitely a (bad) typo.  Thanks for catching this!